packages/valory/skills/apy_estimation_abci/behaviours.py

```diff
@@ -1410,42 +1483,25 @@ def async_act(self) -> Generator:
         - Wait until ABCI application transitions to the next round.
         - Go to the next behaviour (set done event).
         """
-        if (
```

Comment:
What's happened to the cycle_reset logic?

Reply:
Added the `start_fresh` flag which we are now checking in order to avoid comparing strings which is more error-prone and harder to read.